### ISSUE TITLE
Tag is not correct detected

### ISSUE THREAD

**Issue by EyesOnlyNet** [NaturalIntelligence/fast-xml-parser#15]
Hi, i faced an issue regarding the detection of an custom tag.

Given:
```
<?xml version="1.0"?>
<tag.1>label 1</tag.1>
<tag.2>label 2</tag.2>
```

Result:
```
{
    "tag": [
        "label 1",
        "label 2"
    ]
}
```

Expected:
```
{
    "tag1": "label 1",
    "tag2": "label 2",
}
```

Is this expected behaviour? Because W3 said that the "." can be part of the tag name: https://www.w3.org/TR/2008/REC-xml-20081126/#d0e804

I suggest to enhance the `tagsRegx` with the allowed characters: `.`, `:`, `_`, `-`. (in this order)

**Comment by amitguptagwl** [comment]
Ummm, m not sure. Lemme check if it is the case, I'll change it tomorrow
only.

**Comment by amitguptagwl** [comment]
fixed in 2.5.1

**Comment by EyesOnlyNet** [comment]
Great, thank you!
